pipeline/src/transformers/addressables/exhibitionHighlightTour.ts

```diff
@@ -21,11 +22,7 @@ export const transformAddressableExhibitionHighlightTour = (
     : undefined;
   const exhibitionTitle = exhibitionTitleField && asTitle(exhibitionTitleField);
   const introText = data.intro_text && asText(data.intro_text);
-  const primaryImage = relatedExhibition
-    ? relatedExhibition.data.promo?.[0]?.primary
-    : undefined;
-
-  const promoCaption = primaryImage?.caption && asText(primaryImage.caption);
+  const promoCaption = primaryImageCaption(relatedExhibition?.data.promo);
```

Comment:
Would it be sensible to align them all to be `promoCaption` (also named `description` and `caption` in other transformers).